_docs/reviewing_pr.md

```diff
@@ -22,8 +22,9 @@ one of our modules:
 * In the majority of cases, variables shouldn't be accessed via topscope: $::modulename::$param. Instead do: $modulename::$param
 * Are datatypes from stdlib used? Ensure that lowest supported stdlib version is 4.18.0 (This is the first version that supports Puppet 5). Check if a newer version introduced the used datatype
 * Are hiera yaml files added for data-in-modules? Ensure that the data is compatible with [hiera 5](https://puppet.com/docs/puppet/5.3/hiera_migrate.html#use-cases-for-upgrading-to-hiera-5). Static data that is equal across every supported operating system must stay in the init.pp, it shouldn't be moved to a common.yaml due to [puppet-strings issue #250](https://github.com/puppetlabs/puppet-strings/issues/250).
-* Are there new params with datatype Hash or Array? If possible, they should default to empty Hash/Array instead of undef. You can also enforce the datastructure like Array[String[1]]
-* Are there new params with datatype Boolean? The default value is a tricky decision which needs careful reviewing. Sometimes a True/False is the better approach, sometimes undef
+* Are there new parameters with datatype Hash or Array? If possible, they should default to empty `Hash` or `Array` instead of `undef`. You can also enforce the datastructure like `Array[String[1]]` (that's recommended if possible)
```

Comment:
Do we want to recommend fully realized data structures if it's obvious what they should be, or type aliases for more complex things?  so Hash[String, Hash[String, Array[Any]]] for most create_resources collectors

Reply:
good catch. I added:
`* Is the new datatype quite complex (like `Hash[String[1], Hash[String[1], Array[Any]]]`? Create a custom datatype for it.`